plugins/producer_plugin/producer_plugin.cpp

```diff
@@ -428,6 +428,7 @@ class producer_plugin_impl : public std::enable_shared_from_this<producer_plugin
          } catch ( boost::interprocess::bad_alloc& ) {
             chain_plugin::handle_db_exhaustion();
          } catch( const fc::exception& e ) {
+            fc_dlog(_log, "exception during push_block: ${ex}", ("ex", e.to_detail_string()));
```

Comment:
In normal operation this was being reported as an error to the default log  as a result of `handle_error(e)`. 

If we need to also log this to `_log` we should do so in `handle_error` as well so that it will properly report all non-fatal exceptions that are thrown.   

It seems like this may just be an artifact of debugging that could be removed?

Reply:
Thanks for pointing this out. I think we should just remove it as the original 2.1 RC1.